src/main/resources/templates/task/task.html

```diff
@@ -0,0 +1,161 @@
+                            </div>
+                            <span>
+                      <button type="button" class="remove-button" onclick="removeList(this)"><div class="button-text">Remove Task</div></button>
+                    </span>
+                    </li>
+                    <li>
+                        <div class="task-container">
+                            <div class="task-bank-check">
+                                <label class="container">
+                                    <input type="checkbox" class="task-checkbox" value="Task">
+                                    <span class="checkmark"></span>
+                                </label>
+                                <span class="task-bank-task"><h3>r - Audi 2015</h3></span>
+                            </div>
+                            <span>
+                        <button type="submit" class="remove-button" onclick="removeList(this)"><div class="button-text">Remove Task</div></button>
+                      </span>
+                    </li>
+                </ul>
+            </div>
+            <br>
+            <div class="change-job-status">
+                <button type="submit" class="default-button"><img src="../Assets/black-plus.svg"><div class="button-text">Add New Task</div></button>
+            </div>
+        </form>
+    </div>
+</div>
+
+<div id="mySidebar" class="sidebar">
+    <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">×</a>
+    <div class="side-bar-nav">
+        <img onclick="growDiv();" src="../Assets/cardot.svg">
+        <a onclick="growDiv();" href="#" class="selected">Job Management</a>
+        <img class="chevron" onclick="growDiv();" src="../Assets/downarrow.svg">
+    </div>
+    <div class="job-management_nav" id="grow">
+        <div>
+            <img src="../Assets/creating-job.svg">
+            <a href="">Create Job</a>
+        </div>
+        <div class="">
+            <img src="../Assets/job-bank.svg">
+            <a onclick="growJobBank();">Job Bank</a>
+            <img onclick="growJobBank();" class="job-bank_chevron" src="../Assets/downarrow.svg">
+        </div>
+        <div class="job-management_create-job_taskbank" id="grow_taskbank">
+            <div>
+                <img src="../Assets/magnifying.svg">
+                <a href="">Search Job</a>
+            </div>
+        </div>
+        <div>
+            <img src="../Assets/checkout.svg">
+            <a href="">Checkout Job</a>
+        </div>
+    </div>
+    <div class="side-bar-nav">
+        <img src="../Assets/archived.svg">
+        <a href="#">Archived Jobs</a>
+        <img src="">
+    </div>
+    <div class="side-bar-nav">
+        <img src="../Assets/schedule.svg">
+        <a href="#">Schedule</a>
+        <img src="">
+    </div>
+    <div class="side-bar-nav">
+        <img src="../Assets/creating-customer.svg">
+        <a href="#">Creating Customer</a>
+        <img src="">
+    </div>
+</div>
```

Comment:
Thymeleaf tags were not applied on this page + href and src attributes are pointing to old file structure.

Reply:
True, I have corrected this